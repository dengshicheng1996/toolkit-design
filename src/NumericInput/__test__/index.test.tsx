import * as React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import { NumericInput } from '..'

describe('NumericInput', () => {
  it('snapshot', () => {
    const component = shallow(
      <NumericInput className="a" keypadClassName="b" data-y="111" />,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })
})
