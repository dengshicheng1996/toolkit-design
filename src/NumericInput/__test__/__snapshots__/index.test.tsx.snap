// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumericInput snapshot 1`] = `
<div
  className="amd-numeric-input amd-numeric-input-left a"
>
  <Memo([object Object])
    autoFocus={false}
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    value=""
  >
    <div
      className="amd-numeric-input-fake-placeholder"
    />
    <div
      aria-label=""
      className="amd-numeric-input-fake"
      data-y="111"
      role="textbox"
    />
  </Memo([object Object])>
</div>
`;
